### DISCUSSION TITLE
Clean uninstall after appimage run?

### DISCUSSION THREAD

**Discussion by yarko** [beakerbrowser/beaker#1607] [General]
I did a quick appimage run on Ubuntu, and then tried to edit my profile - no end to problems. Then tried downloading 2nd image, woah-mess!

Before I file bugs, what's good way to clean-slate uninstall any "mess", and try for reporting?

**Comment by yarko** [comment]
...answering my own question, I removed my $HOME/.hyperdrive directory - it looks like a "fresh startup" now;   if there is anything else I should cleanup which I might have missed, please let me know.

**Reply by pfrazee** [reply]
If you ever delete ~/.hyperdrive, you also need to delete your beaker user data. In linux, that should be under `~/.config/Beaker Browser` I believe
